### PR TITLE
Adds conception ticket standard

### PR DESCRIPTION
**Pull request by quentinf00**

### CODE REVIEW

File: data-engineering/How-to-use-conception-tickets.s.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+## Why:
+
+ - To detect problem earlier during the implementation of a ticket
+ - To ensure vsibility on what is done during the implementation of a ticket
+
+This can be a tool to reduce the lead time on features with uncertainties, the conception phase can be used to prioritize kanban ticket into you sprint
+
+
+## Checks
```
Comment (flavianh) [inline data-engineering/How-to-use-conception-tickets.s.md:14]: Sur les control points tu peux mettre des emojis comme ici : http://blog.m33.network/2016/09/prepare-your-ceremony-to-make-it-short-and-tactac/

File: data-engineering/How-to-use-conception-tickets.s.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+ - I have detailed the ticket in tasks smaller than 15min
+ - The tasks have been challenged by another dev of the team
+ - The dev tracks the time spent on each tasks using this tool: https://theodo.github.io/workflow-monitor
```
Comment (flavianh) [inline data-engineering/How-to-use-conception-tickets.s.md]: Et si t'as pas Trello comme sur RSolve ?

File: data-engineering/How-to-use-conception-tickets.s.md (hunk 1 above)
Comment (quentinf00) [reply data-engineering/How-to-use-conception-tickets.s.md:14]: Oki :)

File: data-engineering/How-to-use-conception-tickets.s.md (hunk 2 above)
Comment (quentinf00) [reply data-engineering/How-to-use-conception-tickets.s.md]: Good point, you can do your conception phase on your jira card, but you need to be extra rigorous about time tracking on each task. You can ask some co dev to help you track your time

**Pull request merged**